code/cansat_code/thermometer/temperature_sensor.py

```diff
@@ -21,32 +23,22 @@ def initiazlize(self):
         GPIO.setup(self.RCpin, GPIO.IN, pull_up_down=GPIO.PUD_DOWN)
 
     def getSensorData(self):
-        return ('jj', 'cansat')
-        RHW, TW = Adafruit_DHT.read_retry(Adafruit_DHT.DHT11, self.DHTpin)
-        return str(RHW, TW)
-
-    def RCtime(self):
-        return '69'
-        LT = 0
-
-        if GPIO.input(self.RCpin) is True:
-            LT += 1
-        return str(LT)
+        self.read_index += 1
+        return (str(r(10, 100)), str(r(10, 100)))
+        hum, temp = Adafruit_DHT.read_retry(Adafruit_DHT.DHT11, self.DHTpin)
+        return str(temp, hum)
 
     def loop(self):
         print("DHT11 test program by JJCanSat team")
         print('            Starting...')
 
         while True:
             try:
-                print('chuj')
-                RHW, TW = self.getSensorData()
-                print('fuck'+ RHW, TW)
-                LT = self.RCtime()
-                print()
-                print(RHW + ' -- ' + TW + ' -- ' + LT)
+                hum, temp = self.getSensorData()
+                print(str(self.read_index) + ':\t' + temp + '*C | ' + hum + '%')
```

Comment:
https://www.loggly.com/blog/4-reasons-a-python-logging-library-is-much-better-than-putting-print-statements-everywhere/ - używamy logging zamiast printów jak chcemy już coś wywalać na ekran